### DISCUSSION TITLE
Projection Feedback

### DISCUSSION THREAD

**Discussion by Whiskeysaurus** [helgoboss/helgobox#1221] [General]
Apologies for the stupid question, but looking through Issues and Patchnotes, I haven't been able to piece together where this is at with each of the various platforms.

I'm aware of the projection reaper app, the webapp, and the ReaLearn Companion Android app.
I've assumed realtime control feedback was the point of these apps, but I can't find any way to make any of them support it. They just show dotted outlines around the controls. Am I missing a step in setting them up?

(Note, I haven't been able to get any of my android devices to accept the security cert for ReaLearn Companion, so I haven't tried on the android app)

**Comment by helgoboss** [comment]
The Companion app will disappear soon, both the web and Android webapp. The Helgobox App with its Projection view will be the successor. It doesn't support value feedback yet. It just shows which control is mapped to which target ... which was the original purpose of Projection and is still the main feature. It does this only if you set it up correctly.
